src/openforms/emails/models.py

```diff
@@ -19,12 +19,7 @@ def set_for_form(self, form: Form, data: Optional[dict]):
 
 
 class ConfirmationEmailTemplate(models.Model):
-    """Template of email to be sent on completion of submission
-
-    Subject and content fields are validated independently. If the confirmation email
-    is specific to a particular form, both fields must be filled in or `is_usable` will
-    return `False`.
-    """
+    """Template of email to be sent on completion of submission"""
```

Comment:
the `is_usable` method can be removed, no?

Reply:
I removed it and updated some test that used it